python/ray/runtime_context.py

```diff
@@ -254,12 +254,32 @@ def get_actor_id(self) -> Optional[str]:
         if self.worker.mode != ray._private.worker.WORKER_MODE:
             logger.warning(
                 "This method is only available when the process is a "
-                "worker. Current mode: {self.worker.mode}"
+                f"worker. Current mode: {self.worker.mode}"
             )
             return None
         actor_id = self.worker.actor_id
         return actor_id.hex() if not actor_id.is_nil() else None
 
+    def get_actor_name(self) -> Optional[str]:
+        """Get the current actor name of this worker.
+
+        This shouldn't be used in a driver process.
+        The name is in string format.
+
+        Returns:
+            The current actor name of this worker.
+            Return empty string if there's no actor name or if it's a driver
```

Comment:
This docstring is inconsistent. 

`If it is a driver == if it is not an actor`

but it says 

if it is a driver -> empty string
if it is not an actor -> None.

Can you update the docstring to

```
If a current worker is an actor, and if actor name doesn't exist, it returns an empty string.

If a current worker is not an actor, it returns None. 
```